index.html

```diff
@@ -0,0 +1,747 @@
+                
+            <div>
+                    <ul>
+                <li>
+                    <button class="Subscrib-now" type="button">SUBSCRIBE NOW
+                    </button>
+                </li>
+                <li>                    
+                        <button class="Log In" type="button">LOG IN</button>
+                    
+                </li>
+            
+            </ul>
+            </div>
+        </div>
+        </nav>
+    </header>
+     <main>
+        <div class="header">
+            <h1 class="heading">
+                Space Ripples Reveal Big Bang’s Smoking Gun
+            </h1>
+        </div>
+            <div >
+                <figure class="banner">
+                    <img src="assets/images/Alanglut.jpg" alt="Alan Guth was one of the first 
+                    physicists to hypothesize the 
+                    existence of inflation, which explains how the 
+                    universe expanded so uniformly and so quickly in 
+                    the instant after the Big Bang 13.8 billion years ago." width="960" >
```

Comment:
Please, remove inline styling, it is considered a bad practice, move it to your CSS files.